File: Leetcode/Leet1672.java
//https://leetcode.com/problems/richest-customer-wealth/
public class Leet1672 {
    public int maximumWealth(int[][] accounts) {
        int max = 0;
        for(int i = 0; i<accounts.length; i++){
            int temp = 0;
            for(int j = 0; j<accounts[i].length; j++){
                temp += accounts[i][j];
            }
            if(temp > max){
                max = temp;
            }
        }return max;
    }
}

File: Leetcode/Leet1672.py
#https://leetcode.com/problems/richest-customer-wealth/
class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        maX = 0
        for i in accounts:
            if sum(i) > maX:
                maX = sum(i)
        return maX
        